server/hydrogen-render/2-render-hydrogen-to-string-fork-script.js

```diff
@@ -88,10 +89,10 @@ process.on('message', async (renderOptions) => {
     });
   } catch (err) {
     // We need to wait for the error to completely send to the parent
-    // process before we throw the error again and exit the process.
+    // process before we exit the process.
     await serializeError(err);
 
-    // Throw the error so the process fails and exits
-    throw err;
+    // Fail the process and exit
+    process.exit(1);
```

Comment:
This is an important change! The previous code didn't actually fail and exit the process for us and it's unclear to me if it ever did.

This works to show us a `TypeError: ...` immediately for example instead of having the render process timeout and mask the error.